.github/workflows/matchers/black.json

```
@@ -5,7 +5,7 @@
       "severity": "error",
       "pattern": [
         {
-          "regexp": "^(would reformat) (.+)$",
+          "regexp": "^(Run `black \.`.+): (.+)$",
```

```suggestion
          "regexp": "^(Run `black \\.`.+): (.+)$",
```

Do we need one more `\`?